config: add Vuforia

MeloLawson opened the pull request:
add Vuforia package, delete AR Foundation
modify the scene for Vuforia Engine and add coffee machine model target

**note** : add a .gitattributes to authorize large size files (vuforia package .tgz necessary for apk build)

closed #53

MeloLawson committed:
config: add Vuforia

add Vuforia package, delete AR Foundation
modify the scene for Vuforia Engine and add coffee machine model target

MelvinCou commented:
![image](https://github.com/user-attachments/assets/bb500d7f-9fde-4104-92e5-1bc5ad93f4f5)

That's why CI does not pass https://github.com/MelvinCou/Coffee-Show-Up/actions/runs/12199691238/job/34034121351#step:6:1259

Maybe push the file?

MeloLawson committed:
fix: add vuforia package tgz

add vuforia tgz necessary for the apk build
also add .gitattributes to authorize large size files in commit

MelvinCou commented:
LFS is not currently enabled in the CI.

https://github.com/MelvinCou/Coffee-Show-Up/blob/2995888d9b49652888b8e970d5fe48a62111ad35/.github/workflows/unity.yml#L19-L20

```diff
\- name: Checkout repository
   uses: actions/checkout@v4
+  with:
+    lfs: true
```

MeloLawson committed:
ci: enable lfs in ci

MelvinCou commented:
🚀 Android APK available at: https://github.com/MelvinCou/Coffee-Show-Up/actions/runs/12283417816

MelvinCou commented:
Let's hope we won't have too much huge files!

![image](https://github.com/user-attachments/assets/134793c1-345e-44bf-8b48-79d6f019b6a1)
